routes/api/settings/index.js

```diff
@@ -4,19 +4,21 @@ const Setting = require('../../../models/setting');
 const router = express.Router();
 
 router.get('/', (req, res, next) => {
-  Setting
-    .getSettings()
-    .then(settings => {
-      res.json(settings);
-    })
-    .catch(next);
+  Setting.retrieve().then((settings) => {
+    res.json(settings);
+  })
+  .catch((err) => {
```

Comment:
This seems more verbose than the previous implementation of catch, why the change?